build/config/RepoUtilData.json

```diff
@@ -1,6 +1,16 @@
 ﻿{
   "fixedPackages": {
     "Microsoft.VSSDK.BuildTools": [ "14.3.25420" ],
+    "Microsoft.VisualStudio.CoreUtility": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Editor": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.Data": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.Logic": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.UI": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.UI.Wpf": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Utilities": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Shell.Framework": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Shell.14.0": ["14.0.23205"],
+    "Microsoft.VisualStudio.ComponentModelHost": ["14.0.23205", "15.0.26014-alpha"],
```

Comment:
No.  This list is saying essentially two things 

1. What package versions are explicitly meant to be different from the rest of the repo 
2. What package versions should I not upgrade when doing a `change` command

Basically it needs to track the outliers.  In this case the outlier is Dev14 and is the only thing we need to track.